gobblin-service/src/main/java/org/apache/gobblin/service/modules/scheduler/GobblinServiceJobScheduler.java

```diff
@@ -466,8 +466,9 @@ public synchronized void scheduleJob(Properties jobProps, JobListener jobListene
   @Override
   protected void logNewlyScheduledJob(JobDetail job, Trigger trigger) {
     Properties jobProps = (Properties) job.getJobDataMap().get(PROPERTIES_KEY);
-    log.info(jobSchedulerTracePrefixBuilder(jobProps) + "nextTriggerTime: {} - Job newly scheduled",
-         asUTCEpochMillis(trigger.getNextFireTime()));
+    log.info(jobSchedulerTracePrefixBuilder(jobProps) + "nextTriggerTime: {} localConversionTriggerTime:{} - Job newly "
```

Comment:
shouldn't this be named localConversion*Next*TriggerTime ?

nit: rather than the prefix "local conversion", which suggests how we calculate it, I'd opt instead for "localTZ" or "localZoned", which names what it is